boards/pba-d-01-kw2x/include/board.h

```diff
@@ -60,80 +60,32 @@ extern "C"
  * @name Macro for button S1/S2.
  * @{
  */
-#define BTN0_PORT           PORTD
+#define BTN0_PORT           (PORTD)
 #define BTN0_PIN            GPIO_PIN(PORT_D, 1)
-#define BTN0_MODE           GPIO_IN_PU
+#define BTN0_MODE           (GPIO_IN_PU)
 /** @} */
 
 /**
  * @name Macro for capacitive sensor button.
  * @{
  */
-#define BTN1_PORT           PORTC
+#define BTN1_PORT           (PORTC)
 #define BTN1_PIN            GPIO_PIN(PORT_C, 6)
-#define BTN1_MODE           GPIO_IN
+#define BTN1_MODE           (GPIO_IN)
 /** @} */
 
 /**
  * @name KW2XRF configuration
  *
  * {spi bus, cs pin, int pin, spi speed,}
- */
-#define KW2XRF_PARAMS_BOARD  {.spi = SPI_DEV(1), \
-                              .spi_clk = SPI_CLK_10MHZ, \
-                              .cs_pin = GPIO_PIN(KW2XDRF_PORT, KW2XDRF_PCS0_PIN), \
-                              .int_pin = GPIO_PIN(KW2XDRF_PORT, KW2XDRF_IRQ_PIN) }
-#define KW2XRF_SHARED_SPI   0
-/** @}*/
-
-/**
- * @name Define the interface for the HDC1000 humidity sensor
- * @{
- */
-#define HDC1000_I2C         (I2C_DEV(0))
-#define HDC1000_ADDR        (0x43)
-/** @} */
-
-/**
- * @name Define the interface for the MAG3110 magnetometer sensor
- * @{
- */
-#define MAG3110_I2C         (I2C_DEV(0))
-#define MAG3110_ADDR        (0x0E)
-/** @} */
-
-/**
- * @name Define the interface for the MMA8652 tri-axis accelerometer sensor
  * @{
  */
-#define MMA8652_I2C         (I2C_DEV(0))
-#define MMA8652_ADDR        (0x1D)
-/** @} */
-
-/**
- * @name Define the interface for the MPL3115A2 pressure sensor
- * @{
- */
-#define MPL3115A2_I2C       (I2C_DEV(0))
-#define MPL3115A2_ADDR      (0x60)
-/** @} */
-
-/**
- * @name Define the interface for the TCS3772 light sensor
- * @{
- */
-#define TCS37727_PARAMS     { .i2c   = I2C_DEV(0), \
-                              .addr  = 0x29, \
-                              .atime = TCS37727_PARAM_ATIME }
-/** @} */
-
-/**
- * @name Define the interface for the TMP006 IR-Termopile sensor
- * @{
- */
-#define TMP006_I2C          (I2C_DEV(0))
-#define TMP006_ADDR         (0x41)
-/** @} */
+#define KW2XRF_PARAM_SPI           SPI_DEV(1)
+#define KW2XRF_PARAM_SPI_CLK       (SPI_CLK_10MHZ)
```

Comment:
Just to be consistent you might want to remove the parenthesis on this define.

Reply:
Here and below btw.

Reply:
Consistent with what ? I just tested a git grep git grep 'SPI_CLK_10MHZ' both syntaxes (with and without parenthesis) are used. I tend to prefer with parenthesis. Is there a consensus for that ?